integration_tests/pages/page.js

```diff
@@ -4,7 +4,7 @@ module.exports = (name, pageObject = {}, axeTest = true) => {
     cy.checkA11y()
   }
 
-  const checkOnPage = () => cy.get('h1').contains(name)
+  const checkOnPage = () => cy.get('h1').contains(name) || cy.get('title').contains(name)
   const logout = () => cy.get('[data-qa=logout]')
```

Comment:
The confirmation page has no <h1> header - so added a check for the <title> here too.